src/main/kotlin/gg/essential/elementa/constraints/CramSiblingConstraint.kt

```diff
@@ -109,10 +109,19 @@ class CramSiblingConstraint(padding: Float = 0f) : SiblingConstraint(padding) {
 
         val sibling = component.parent.children[index - 1]
 
-        if (sibling.getRight() + component.getWidth() + padding <= component.parent.getRight()) {
+        if (sibling.getRight() + component.getWidth() + padding <= component.parent.getRight() + precisionAdjustmentFactor) {
             return 0F
         }
 
         return padding
     }
+
+    companion object {
```

Comment:
Should be `private`.
Otherwise that's a new part of the ABI and that's the reason [the build fails](https://ci.essential.gg/buildConfiguration/Elementa_Build/3236?showRootCauses=false&expandBuildChangesSection=true&expandPull+Request+Details=true&expandBuildProblemsSection=true).